scipy/stats/tests/test_stats.py

```diff
@@ -3001,27 +3001,25 @@ def _assert_equal(self, actual, expect, *, shape=None, dtype=None):
             dtype = expect.dtype
         assert actual.dtype == dtype
 
-    @pytest.mark.parametrize('m, c, expect', [
-        (1, 0, 2.5),
-        (1, 2.5, 0),
-        (1, 2, 0.5),
-        (2, 0, 7.5),
-        (2, 1, 3.5),
-        (3, 1, 9.0),
-        (3, 2, 2.0)
```

Comment:
The source of the reference values was not listed. If they are calculated by hand, why not let the computer calculate them so that they do not need to be calculated by hand?